bench/Chainweb/Pact/Backend/ForkingBench.hs

```diff
@@ -379,7 +379,7 @@ cid :: ChainId
 cid = someChainId testVer
 
 testVer :: ChainwebVersion
-testVer = FastTimedCPM petersonChainGraph
+testVer = slowForkingCpmTestVersion petersonChainGraph
```

Comment:
Can you explain this change to me?

Reply:
Historically we special-cased the fork heights of `FastTimedCPM petersonChainGraph` to be slower, so any other chain graph gives faster fork heights. Confusing, eh? With this PR, you ask for the slower or faster fork heights, and you ask for the chain graph separately.

tl;dr:
`FastTimedCPM petersonChainGraph` corresponds to `slowForkingCpmTestVersion petersonChainGraph`, whereas `FastTimedCPM g` for `g /= petersonChainGraph` corresponds to `fastForkingCpmTestVersion petersonChainGraph`.

Reply:
For example:
```hs
pact420Upgrade :: ChainwebVersion -> BlockHeight -> Bool
pact420Upgrade Mainnet01 = (>= 2_334_500) -- 2022-01-17T17:51:12
pact420Upgrade Testnet04 = (>= 1_862_000) -- 2022-01-13T16:11:10
pact420Upgrade Development = (>= 90)
pact420Upgrade (FastTimedCPM g) | g == petersonChainGraph = (>= 5)
pact420Upgrade _ = const True
```